README.md

```diff
@@ -252,56 +252,11 @@ Its value is used to match the role that impersonates the request, against the r
 If the current key does not match, the less specific key is looked up and matched following the steps 1 to 4 above, up until the `default` key.
 Otherwise, the operation invocation is denied.
 
+# Monitoring
 
-## Prometheus Operator for Fuse
-
-Monitor your Fuse applications and take advantage of the built-in instrumentation with the prometheus-operator.
-
-You can run the following instructions to deploy the prometheus-operator in your Fuse namespace:
-
-First, install the CustomResourceDefinitions necessary for running the prometheus-operator.   Note that you will need to be logged in as a user with cluster admin permissions.  
-
-```
-$ oc login -u system:admin
-$ oc create -f fuse-prometheus-crd.yml
-```
-
-Then, install the prometheus-operator to your namespace:
-
-```
-$ oc process -f fuse-prometheus-operator.yml  -p NAMESPACE=<YOUR NAMESPACE> | oc create -f -
-```
-
-Finally, tell it to monitor your fuse application:
-
-```
-$ oc process -f fuse-servicemonitor.yml -p NAMESPACE=<YOUR NAMESPACE> -p FUSE_SERVICE_NAME=<YOUR FUSE SERVICE> | oc create -f -
-```
-
-Note that the `NAMESPACE` and `FUSE_SERVICE_NAME` parameters must be specified.
-
-
-## Fuse Apicurito
```

Comment:
Is it ok to remove the Apicurito section as well?  It doesn't seem to be related to monitoring.